akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2FrameSending.scala

```diff
@@ -0,0 +1,46 @@
+package akka.http.impl.engine.http2
+
+import java.nio.ByteOrder
+
+import akka.http.impl.engine.http2.FrameEvent.{ ContinuationFrame, HeadersFrame, PriorityFrame, Setting, SettingsFrame, WindowUpdateFrame }
+import akka.http.impl.engine.http2.Http2Protocol.{ ErrorCode, Flags, FrameType, SettingIdentifier }
+import akka.http.impl.engine.http2.framing.FrameRenderer
+import akka.util.{ ByteString, ByteStringBuilder }
+
+trait Http2FrameSending {
```

Comment:
Extracted from `TestSetupWithoutHandshake` in `Http2ServerSpec`.


While the `TestSetupWithoutHandshake` itself is different between client and server, the utilities for creating and sending frames can be shared (though they will represent incoming 'request frames' for the server test and incoming 'response frames' for the client test)